README.md

```diff
@@ -61,6 +58,29 @@ Enterprise servers can be configured via the `"deephaven.enterpriseServers"` set
 
 ![Enterprise Server Settings](./docs/assets/dhe-settings.gif)
 
+## SSL Certificates
+Deephaven servers using self-signed certificates or internal CA's will require configuring VS Code to trust the signing certificate.
+
+1. Save the signing certificate in PEM format somewhere on the machine running VS Code. Multiple certificates can be concatenated together in the same file if there are multiple certs that need to be configured.
+1. Set the `NODE_EXTRA_CA_CERTS` environment variable to the path of the signing certificate.
+   
+   On Mac / Linux, you set the env variable or if you'd like for it to persist, you can export it from an appropriate config file for your shell.
+   ```sh
+   export NODE_EXTRA_CA_CERTS=/path/to/cert.pem
+   ```
+
+   On Windows, you can use `set` to set the variable in your current shell, or `setx` to persist it.
+
+   ```sh
+   setx NODE_EXTRA_CA_CERTS C:\Path\To\cert.pem
+   ```
+   > Note that paths in env variables should not be wrapped in quotes on Windows.
+1. Start VS Code in a shell that has the `NODE_EXTRA_CA_CERTS` variable set.
```

Comment:
```suggestion
3. Start VS Code in a shell that has the `NODE_EXTRA_CA_CERTS` variable set.
```

Reply:
Same here, Markdown supports numbering as 1. 1. 1. ... and auto increments. Makes it easier to insert / re-order in the future without modifying every line.